cmd/svcat/instance/get_cmd.go

```diff
@@ -37,6 +38,7 @@ func NewGetCmd(cxt *command.Context) *cobra.Command {
 		Short:   "List instances, optionally filtered by name",
 		Example: `
   svcat get instances
+  svcat get instances --all-namespaces
```

Comment:
This feels like a pretty verbose flag, what if we make the default for `-n` to be "" and in a sense, the lack of specifying a namespace would be all namespaces. If you want the default namespace you have to say `-n default`.

Reply:
What say you @carolynvs ?

Reply:
That would be a behavior difference on a shared component between svcat and kubectl which we avoid to maintain consistent behavior between using svcat alone and as a kubectl plugin.

When a namespace is not set in the current context, kubectl uses "default", and to query all namespaces, kubectl has the `--all-namespaces` flag. This PR brings svcat in-line with kubectl, and enables us to respect the `--all-namespaces` flag when it is passed to us by kubectl in plugin mode.

Once this goes in, you can continue to use `-n=""` to get all namespaces if you prefer a shorter syntax. 😁

Reply:
Yeah, I would stick with the behavior kubectl uses as well - and it uses --all-namespaces for several commands.

Reply:
ok thanks for the clarification!!